config/apis.ts

```diff
@@ -6,6 +6,30 @@ const addressValidation: KeyAuthAPI = {
   name: 'Address Validation API',
 };
 
+const appealsStatus: KeyAuthAPI = {
+  acl: 'appeals',
+  key: 'appeals',
+  name: 'Appeals Status API',
+};
+
+const clinicalHealth: OAuthAPI = {
+  authzEndpoint: process.env.AUTHZ_SERVER_CLINICAL_FHIR,
```

Comment:
would suggest adding a default value since this config doesn't actually exist. something like `?? 'UNKNOWN'` + a comment so people understand what's going on here.

Reply:
added in latest commit!